README.md

```diff
@@ -33,33 +33,50 @@ But there's more.
 You can cache results by configuring the resolver to use a `CachedExecutor`:
 
 ```php
-    $loop = React\EventLoop\Factory::create();
-    $factory = new React\Dns\Resolver\Factory();
-    $dns = $factory->createCached('8.8.8.8', $loop);
+$loop = React\EventLoop\Factory::create();
+$factory = new React\Dns\Resolver\Factory();
+$dns = $factory->createCached('8.8.8.8', $loop);
+
+$dns->resolve('igor.io')->then(function ($ip) {
+    echo "Host: $ip\n";
+});
 
-    $dns->resolve('igor.io')->then(function ($ip) {
-        echo "Host: $ip\n";
-    });
+...
 
-    ...
+$dns->resolve('igor.io')->then(function ($ip) {
+    echo "Host: $ip\n";
+});
 
-    $dns->resolve('igor.io')->then(function ($ip) {
-        echo "Host: $ip\n";
-    });
-    
-    $loop->run();
+$loop->run();
 ```
 
 If the first call returns before the second, only one query will be executed.
 The second result will be served from cache.
 
-## Todo
+## Install
+
+The recommended way to install this library is [through Composer](http://getcomposer.org).
+[New to Composer?](http://getcomposer.org/doc/00-intro.md)
+
+This will install the latest supported version:
+
+```bash
+$ composer require react/dns:~0.4.0
```

Comment:
Personally I have a medium strong opinion about the right composer instructions. But just wanted to point out a maybe better way of doing it. The `latest supported version` works in favor of doing it this way to be honest, I'm good either way.